build.gradle.kts

```diff
@@ -1,10 +1,10 @@
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
-val javaVersion = JavaVersion.VERSION_11
+val javaVersion = JavaVersion.VERSION_22
```

Comment:
I'm not sure we should jump straight to Java 22, especially if we're not using newer features. This is mostly because of the backend module which servers might be using to compile scripts during cache packing that likely aren't on Java 22.